app/src/main/java/com/babylon/wallet/android/presentation/settings/SettingsItem.kt

```diff
@@ -99,7 +99,12 @@ sealed interface SettingsItem {
     sealed interface Troubleshooting {
         data object AccountRecovery : Troubleshooting
         data object ImportFromLegacyWallet : Troubleshooting
-        data object ContactSupport : Troubleshooting
+        data class ContactSupport(
```

Comment:
I'm not sure if we settled on support email template, this might change and/or be defined in crowdin at some point